internal/objects/daemonset/daemonset_test.go

```diff
@@ -123,8 +123,6 @@ func TestDesiredDaemonSet(t *testing.T) {
 	cfg := objcontour.Config{
 		Name:        name,
 		Namespace:   fmt.Sprintf("%s-ns", name),
-		SpecNs:      "projectcontour",
-		RemoveNs:    false,
```

Comment:
it'd be nice to have this test verify that the DS gets created in `cfg.Namespace`. Similarly for the other `TestDesired*` tests.